packages/vsce/src/SshConfigUtils.ts

```diff
@@ -473,10 +473,7 @@ export class SshConfigUtils {
         return selectedProfile;
     }
 
-    private static async setProfile(
-        selectedConfig: Partial<ISshConfigExt> | undefined,
-        updatedProfile?: string,
-    ): Promise<void> {
+    private static async setProfile(selectedConfig: ISshConfigExt | undefined, updatedProfile?: string): Promise<void> {
```

Comment:
Thanks for considering removing `Partial`

However, there are a few more instances...

![image](https://github.com/user-attachments/assets/d2b990bd-c608-4a91-ba9e-168df21b134d)

---

If we need them for some reason, feel free to ignore this comment 😋